core/provisioning-api/src/main/java/org/apache/syncope/core/provisioning/api/data/AuthProfileDataBinder.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
This class should not be needed, the purpose of databinding is to transform entities in TOs and viceversa. Just use `entityFactory` in the calling place.

Reply:
@mmoayyed what about this?

Reply:
Given we have an AuthProfileTO entity, should this still be removed?

Reply:
Ah, I see you are right, please ignore this.